example_test.go

```diff
@@ -271,3 +272,21 @@ func ExampleUsage() {
 	//   THREE string
 	//     	third parameter
 }
+
+func ExampleUnexportedField(t *testing.T) {
+	conf := struct {
+		Database struct {
+			Host        string `yaml:"host" env:"DB_HOST" env-description:"Database host"`
+			Port        string `yaml:"port" env:"DB_PORT" env-description:"Database port"`
+		} `yaml:"database"`
+		server struct {
+			Host string `yaml:"host" env:"SRV_HOST,HOST" env-description:"Server host" env-default:"localhost"`
+			Port string `yaml:"port" env:"SRV_PORT,PORT" env-description:"Server port" env-default:"8080"`
+		} `yaml:"server"`
+	}{}
+
+	if err := cleanenv.ReadConfig("example/simple_config/config.yml", &conf); err != nil {
+		t.Fatal(err)
+	}
+	t.Logf("%+v", conf)
```

Comment:
The example has to print some output and compare it with [a predefined text comment](https://go.dev/blog/examples). Either change it to do some output with the expected value or change the example to test (I prefer the latter). 

Also, it has to actually check something - like if exported fields are filled and not exported fields are empty.